frontend/src/main/java/puf/frisbee/frontend/view/WaitingView.java

```diff
@@ -29,6 +38,10 @@ public void init(WaitingViewModel waitingViewModel, ViewHandler viewHandler) {
         this.viewHandler = viewHandler;
         this.waitingViewModel = waitingViewModel;
 
+        this.level01BackgroundImage.visibleProperty().bind(this.waitingViewModel.getShowLevel01BackgroundImageProperty());
```

Comment:
Weil wir diese ganze Background Logik doppelt haben, eine grobe Idee (vllt ist sie auch doof):
Eine BackgroundImage View, die in WaitingView und GameView eingebunden wird, und Zugriff auf das Team View Model hat. Durch das Einbinden in GameView und WaitingView wird das init ja jedes Mal neu aufgerufen und es wird der aktuelle Level Status vom Team Model View geholt?

Reply:
Ja, das könnte gehen. Aber lieber auf einem neuen Ticket: #173

Reply:
Morgen wollen wir ja erst mal auch noch keinen Code zeigen.